doc/data/messages/t/too-many-positional-arguments/details.rst

```diff
@@ -1,10 +1,11 @@
-Positional arguments work well for cases where the the use cases are
-self-evident, such as unittest's ``assertEqual(first, second, msg=None)``.
-Comprehensibility suffers beyond a handful of arguments, though, so for
-functions that take more inputs, require that additional arguments be
-passed by *keyword only* by preceding them with ``*``:
-
-.. code-block:: python
+Good APIs don’t have many positional parameters. For almost all API,
+comprehensibility suffers beyond a handful of arguments.
```

Comment:
In case people think API == web APIs, this might be less distracting:

```suggestion
Good function signatures don’t have many positional parameters. For almost all
interfaces, comprehensibility suffers beyond a handful of arguments.
```